Source/WebCore/accessibility/AccessibilityMathMLElement.cpp

```diff
@@ -59,7 +59,7 @@ AccessibilityRole AccessibilityMathMLElement::determineAccessibilityRole()
         return m_ariaRole;
 
     Node* node = m_renderer->node();
-    if (node && node->hasTagName(MathMLNames::mathTag))
+    if (node && WebCore::elementName(node) == ElementName::MathML_math)
```

Comment:
Could use the Node& version of elementName here to avoid duplicating the null-check